common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/client/LwM2mClient.java

```diff
@@ -366,8 +369,7 @@ public String isValidObjectVersion(String path) {
             return String.format("Specified object id %s absent in the list supported objects of the client or is security object!", pathIds.getObjectId());
         } else {
             String verRez = getVerFromPathIdVerOrId(path);
-            if ((verRez != null && !verRez.equals(verSupportedObject)) ||
-                    (verRez == null && !LWM2M_OBJECT_VERSION_DEFAULT.equals(verSupportedObject))) {
+            if (verRez == null || !verRez.equals(verSupportedObject)) {
```

Comment:
Why is this changed?

Reply:
If verSupportedObject==null ? verSupportedObject = LWM2M_OBJECT_VERSION_DEFAULT